src/api/utils/getData.ts

```diff
@@ -0,0 +1 @@
+export const getData = <T>(url: string): Promise<T> => fetch(url).then(response => response.json());
```

Comment:
Делай переносы, не пиши такие длинные строки
```suggestion
export const getData = <T>(url: string): Promise<T> => fetch(url)
  .then(response => response.json());
```